README.md

```diff
@@ -18,54 +18,56 @@ These samples provide a starting point for how to integrate different services u
 
 ## Samples of Docker Compose applications with multiple integrated services
 
-<img src="icon_devenvs.svg" alt="Use with Docker Dev Environments" height="30" align="top"/> Icon indicates Sample is compatible with [Docker Dev Environments](https://docs.docker.com/desktop/dev-environments/)
+<img src="icon_devenvs.svg" alt="Use with Docker Dev Environments" height="30" align="top"/> Icon indicates Sample is compatible with [Docker Dev Environments](https://docs.docker.com/desktop/dev-environments/). The _Open in Docker Dev Environments_ link to open the sample project directly works with Docker Desktop 4.11 and later.
 
 - [`ASP.NET / MS-SQL`](https://github.com/docker/awesome-compose/tree/master/aspnet-mssql) - Sample ASP.NET core application
 with MS SQL server database.
 - [`Elasticsearch / Logstash / Kibana`](https://github.com/docker/awesome-compose/tree/master/elasticsearch-logstash-kibana) - Sample Elasticsearch, Logstash, and Kibana stack.
 - [`Go / NGINX / MySQL`](https://github.com/docker/awesome-compose/tree/master/nginx-golang-mysql) - Sample Go application
-with an Nginx proxy and a MySQL database.&nbsp;<img src="icon_devenvs.svg" alt="Use with Docker Dev Environments" height="30" align="top"/>
+with an Nginx proxy and a MySQL database.&nbsp;<img src="icon_devenvs.svg" alt="Use with Docker Dev Environments" height="30" align="top"/> [Open in Docker Dev Environments](https://open.docker.com/dashboard/dev-envs?url=https://github.com/docker/awesome-compose/tree/master/nginx-golang-mysql)
```

Comment:
What do you think if we use directly the icon and we add a tooltip on it to display the "Open in Docker Dev Environments" message?
```suggestion
with an Nginx proxy and a MySQL database.&nbsp;[<img src="icon_devenvs.svg" alt="Use with Docker Dev Environments" height="30" align="top"/>](https://open.docker.com/dashboard/dev-envs?url=https://github.com/docker/awesome-compose/tree/master/nginx-golang-mysql "Open in Docker Dev Environments")
```